charts/work-package/values.yaml

```diff
@@ -27,6 +27,9 @@ ingress:
     enabled: false
     secretName: ""
 
+serviceaccount:
+  create: false
```

Comment:
You can remove that since we set the default in the library chart.

Reply:
Yes. I haven't updated this PR because I was waiting on this PR (https://github.com/ghga-de/charts/pull/81) to be reviewed.